news/2023/2023-03-02-osumonthly-8.md

```diff
@@ -0,0 +1,253 @@
+
+#### Automated audio spectograms with AxerBot
+
+As you probably know, many Discord bots are created by members of the community in order to make life easier for players or simply connect the osu! database with Discord. [AxerBot](https://osu.ppy.sh/community/forums/topics/1604925?n=1) is one of these bots, created by [Hivie](https://osu.ppy.sh/users/14102976) and [Sebola](https://osu.ppy.sh/users/15821708), and which received an important update this month that will please numerous mappers
+
+The bot now allows creating and displaying an audio file's spectrogram and fix any potential bloating issues, in addition to all its already existing features such as the ability to view many mapping and playing stats or having rich integration with the [BN website](https://bn.mappersguild.com/). These kinds of bots will only impress us with new features making life easier from day to day, and we hope to see new ones soon!
+
+#### LeFeraille's osu! anime
+
+A wonderful discovery this month is [LeFeraille's YouTube channel](https://www.youtube.com/channel/UC8fLmUUi8wyT4dopcIEybpA), and its amazing and funny animations. This month in particular, a video called ["Pippi is a tournament player"](https://www.youtube.com/watch?v=FpyaQE_J548) features Pippi and Default, two friends playing osu! and discussing tournaments while throwing references and jokes left and right. If you have four minutes to dedicate to this little creator, don't hesitate to go and have a look!
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/FpyaQE_J548" allowfullscreen></iframe>
+
+### Notable osu! plays of the month
+
+Notable osu! plays of the month
+
+One milestone? Two milestones? How about three.
+
+Once again, [mrekk](https://osu.ppy.sh/users/7562902) has surprised all of us with what he could do, as he [FC'd the top difficulty](https://osu.ppy.sh/scores/osu/4382393442) of [Nevo](https://osu.ppy.sh/users/7451883)'s rendition of [THE ORAL CIGARETTES - Mou Ii kai?](https://osu.ppy.sh/beatmapsets/807850#osu/1695382), something that many top players have attempted to do, and failed, with the closest attempts before being [aetrna](https://osu.ppy.sh/users/6447454)'s 1-miss attempt from 2.5 years ago and [Rimuru](https://osu.ppy.sh/users/9269034)'s 2 misses the very same month, making mrekk's play a good endgame score with its 10 stars of difficulty induced by 288 BPM streams with HDDT scattered across the map, as well as the huge jumps at the end.
+
+You can witness it for yourself below:
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/u5RLfYX_KCw" frameborder="0" allowfullscreen></iframe>
+
+However, that's only the tip of the iceberg. Although we have had many technical 10-star FCs, mrekk is the first player to ever achieve the 10-star Full Combo medal without using a touch device.
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/u5RLfYX_KCw" frameborder="0" allowfullscreen></iframe>
```

Comment:
```suggestion
<iframe width="100%" height="315" src="https://www.youtube.com/embed/u5RLfYX_KCw" frameborder="0" allowfullscreen></iframe>

And lastly, before mrekk set the score shown above, he had also set a score on THE ORAL CIGARETTES - 5150 [Reflection] with HDDT 4 minutes prior to the 10 stars FC, one of the another map that requires highly-skilled both consistency and speed, and not only it is speed but the length matter at 9.85 stars that makes it harder to achieve the FC. Which, you can also watch it below!

<iframe width="100%" height="315" src="https://www.youtube.com/embed/ulPJlMacvtc" frameborder="0" allowfullscreen></iframe>
```

Reply:
added one last more notable score for standard to complement this section ^^

Reply:
added one more last notable score for standard to complement this section ^^

Reply:
may want to use the direct link instead of the embed link there